inference-engine/tests/functional/plugin/shared/src/behavior/stress_tests.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/stress_tests.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string MultipleAllocations::getTestCaseName(const testing::TestParamInfo<MultipleAllocationsParams>& obj) {
+    LayerTestsUtils::TargetDevice targetDevice;
+    unsigned int allocationsCount;
+
+    std::tie(targetDevice, allocationsCount) = obj.param;
+    std::ostringstream result;
+    result << "targetDevice=" << targetDevice << "_";
+    result << "allocationsCount=" << allocationsCount;
+    return result.str();
+}
+
+void MultipleAllocations::SetUp() {
+    std::tie(targetDevice, m_allocationsCount) = this->GetParam();
+    function = ngraph::builder::subgraph::makeSplitConvConcat();
+}
+
+TEST_P(MultipleAllocations, InferWorksCorrectAfterAllocations) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    ConfigurePlugin();
+
+    InferenceEngine::CNNNetwork cnnNet(function);
+    auto ie = PluginCache::get().ie();
+
+    std::cout << "Load the network " << m_allocationsCount << " times..." << std::flush;
+    for (int i = 0; i < m_allocationsCount; ++i) {
+        ie->LoadNetwork(cnnNet, targetDevice);
+    }
+
+    std::cout << "\nCheck inference.\n";
```

Comment:
Personally, I'm against additional output for a test. You forget about that once test passed and then it just garbage in output that no one doesn't look at. Do you agree?

Reply:
But this is a pretty long test, and it would be nice to point out the part that failed. Besides, it's only a couple of lines.

Reply:
I'd prefer to get rid of logs anyway even if it's small, but it's mostly my personal opinion, so if you find it helpful even if tests passing - ok, resolving